channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/channels/TelephonyChannel.kt

```diff
@@ -45,14 +46,16 @@ class TelephonyChannel(
 
     companion object : JaicpNativeChannelFactory {
         override val channelType = "resterisk"
-        override fun create(botApi: BotApi) = TelephonyChannel(botApi)
+        override fun create(botApi: BotApi, liveChatProvider: JaicpLiveChatProvider): JaicpNativeBotChannel =
+            TelephonyChannel(botApi)
```

Comment:
Why isn't liveChatProvider used?

Reply:
Telephony channel does not support Live Chats